spec/models/country_band_spec.rb

```diff
@@ -15,6 +15,6 @@
 
   it "invalidates band with invalid band id" do
     cb = CountryBand.new(number: 6, iso2: "HELLO")
-    expect(cb).to be_invalid_with_errors(number: ["is not included in the list"])
+    expect(cb).to be_invalid_with_errors(country: ["must exist"])
```

Comment:
Regarding my other comment above, feel free to re-add the previous "number not included in the list" validation.

Reply:
Also, why did the `country` validation that you're testing now not exist/fail before?

Reply:
Added back that error. I just realized the country validation test was already there in the unit test just before this.